typhos/cache.py

```diff
@@ -32,6 +39,14 @@ def get_global_widget_type_cache():
     return _GLOBAL_WIDGET_TYPE_CACHE
 
 
+def get_global_path_cache():
+    """Get the _GlobalDisplayPathCache singleton."""
+    global _GLOBAL_PATH_CACHE
+    if _GLOBAL_PATH_CACHE is None:
+        _GLOBAL_PATH_CACHE = _GlobalDisplayPathCache()
```

Comment:
There are some inconsistencies here and there between calling it a "global display path cache" vs a "global path cache", this tripped me up a bit when trying to read through the code paths

Reply:
Resolved in 6e51a31be01f94cb31d6b0cbd58cb4e5157aece1